src/validators/route_type.rs

```diff
@@ -0,0 +1,26 @@
+use crate::validators::issues::*;
```

Comment:
I don't really like global import, It's difficult to know whend reading the code where the objects are from.

I might be wrong, by I think you can import:
```rust
use crate:validators::issues::{Issue, IssueType, Severity};
```

Reply:
Thanks, fixed!